launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -62,15 +65,23 @@
   <!-- radar detection-->
   <group>
     <push-ros-namespace namespace="radar"/>
-    <include file="$(find-pkg-share tier4_perception_launch)/launch/object_recognition/detection/radar_based_detection.launch.xml">
-      <arg name="output/objects" value="objects"/>
+    <include file="$(find-pkg-share perception_launch)/launch/object_recognition/detection/radar_based_detection.launch.xml">
+      <arg name="input/radar" value="$(var input/radar)"/>
+      <arg name="output/objects" value="far_objects"/>
```

Comment:
[Q] `/perception/object_recognition/detection/radar/far_objects` is not subscribed by any node.
Is this supposed to connect to the node implemented https://github.com/autowarefoundation/autoware.universe/pull/4232 ?

Reply:
I'm sorry, I just noticed. You already wrote this in Notes for reviewers section.